libs/application/templates/login-service/src/shared/types.ts

```diff
@@ -0,0 +1,7 @@
+export type selectCompanySearchField = {
+  searchField: {
+    isat: string
+    nationalId: string
+    label: string
+  }
+}
```

Comment:
This type is duplicated here and in the `CompanySearchFromField` we should try to reuse the type.